OpenRA.Mods.Common/Traits/Transforms.cs

```diff
@@ -97,6 +98,20 @@ public bool CanDeploy()
 			return buildingInfo == null || self.World.CanPlaceBuilding(self.Location + Info.Offset, actorInfo, buildingInfo, self);
 		}
 
+		IEnumerable<Order> ClearBlockersOrders(World world, CPos topLeft)
+		{
+			var allTiles = buildingInfo.Tiles(topLeft).ToList();
+			var adjacentTiles = Util.ExpandFootprint(allTiles, true).Except(allTiles)
+				.Where(world.Map.Contains).ToList();
+
+			var blockers = allTiles.SelectMany(world.ActorMap.GetActorsAt)
+				.Where(a => a.Owner == self.Owner && a != self && a.IsIdle)
+				.Select(a => new TraitPair<IMove>(a, a.TraitOrDefault<IMove>()))
+				.Where(x => x.Trait != null);
```

Comment:
I meant including these as well. All you need to pas in is buildingInfo and world

Reply:
No, the required parameters are many more, which is why I purposefully avoided this complicated way, but here you go.